lottie/src/main/java/com/airbnb/lottie/Content.java

```diff
@@ -3,5 +3,7 @@
 import java.util.List;
 
 interface Content {
+  String getName();
```

Comment:
my understanding is that all content can have a name. just put it here to be consitent